integration_tests/integration/yourCompletedReports.cy.ts

```diff
@@ -25,12 +26,15 @@ context('Your Completed Reports', () => {
   it('pagination should work', () => {
     const manyReportedAdjudications: ReportedAdjudication[] = generateRange(1, 300, _ => {
       return {
+        gender: PrisonerGender.MALE,
         adjudicationNumber: _,
         prisonerNumber: 'A1234AA',
         bookingId: 1,
+        dateTimeOfDiscovery: '2021-11-15T11:30:00',
```

Comment:
I don't think this needs to be here - you've already added it into the incidentDetails object below